.travis.yml

```diff
@@ -0,0 +1,244 @@
+  - os: linux
+    compiler: gcc
+    addons: &9
+      apt:
+        sources:
+        - ubuntu-toolchain-r-test
+        - george-edison55-precise-backports
+        packages:
+        - cmake
+        - cmake-data
+        - liblapack-dev
+        - g++-5
+        - gcc-5
+        - gfortran-5
+    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran-5' BUILD_TYPE='release' BOOSTSTR="--boost-incdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0 --boost-libdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0/stage/lib" BOOSTBUILD=".scripts/build_boost.sh 5" OMP_NUM_THREADS=2
+  - os: linux
+    compiler: gcc
+    addons: *9
+    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran-5' BUILD_TYPE='debug' BOOSTSTR="--boost-incdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0 --boost-libdir=${TRAVIS_BUILD_DIR}/boost_build/boost_1_57_0/stage/lib" BOOSTBUILD=".scripts/build_boost.sh 5" OMP_NUM_THREADS=2
+
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: clang
+#    env: CXX_COMPILER='clang++' C_COMPILER='clang' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode6.4
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='debug'
+#  - os: osx
+#    osx_image: xcode7
+#    compiler: gcc
+#    env: CXX_COMPILER='g++-5' C_COMPILER='gcc-5' Fortran_COMPILER='gfortran' BUILD_TYPE='release'
+install:
+- DEPS_DIR="${TRAVIS_BUILD_DIR}/deps"
+- mkdir -p ${DEPS_DIR} && cd ${DEPS_DIR}
+- |
+  if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+    brew install cmake python
+    brew install boost --with-python
+    brew install boost-python
+  fi
+before_script:
+- cd ${TRAVIS_BUILD_DIR}
+- export CXX=${CXX_COMPILER}
+- export CC=${C_COMPILER}
+- export FC=${Fortran_COMPILER}
+- ${BOOSTBUILD}
+- python setup --cxx=${CXX_COMPILER} --cc=${C_COMPILER} --fc=${Fortran_COMPILER} --type=${BUILD_TYPE} ${BOOSTSTR} --max-am-eri=4 --chemps2=off objdir
+- cd objdir
+script:
+- ../.scripts/travis_build.sh
```

Comment:
The structure of the `.travis.yml` is basically the same as in jturney/ambit@a426dff3abae614ede8f237eacb9fab7e7fe7606 and in PCMSolver/pcmsolver@74aef8140c10b8ea62d477b538560c410328a000 so I guess it should work. Another possible improvement would be to run the build in the `script` section and the tests in the `after_script` section. This should give better error messages, so that you don't have to scroll all the way down in the log. I have never tried this though!